test/UnitTestParquetCpp.chpl

```diff
@@ -17,8 +17,8 @@ proc testReadWrite(filename: c_string, dsetname: c_string, size: int) {
   
   var a: [0..#size] int;
   for i in 0..#size do a[i] = i;
-  
-  if c_writeColumnToParquet(filename, c_ptrTo(a), 0, dsetname, size, 10000, false, 1, errMsg) < 0 {
+
+  if c_writeColumnToParquet(filename, c_ptrTo(a), 0, dsetname, size, 10000, 1, false, errMsg) < 0 {
```

Comment:
were these 2 params switched intentionally?

Reply:
Also I think this is why the CI is failing, cause this would also need to be updated here
https://github.com/Bears-R-Us/arkouda/blob/1193c8a1daf632445c28a04734ad7aaaa31bf542/test/UnitTestParquetCpp.chpl#L21

Reply:
I think this is why the CI is failing cause this doesn't match the function signature
https://github.com/Bears-R-Us/arkouda/blob/1193c8a1daf632445c28a04734ad7aaaa31bf542/src/ArrowFunctions.h#L46-L48

Reply:
wait this does match?? okay nevermind me, im confused lol

Reply:
I don't know what I was thinking when I did this, I wish I had an explanation!